common/lib/xmodule/xmodule/progress.py

```diff
@@ -116,12 +116,15 @@ def __ne__(self, other):
         return not self.__eq__(other)
 
     def __str__(self):
-        ''' Return a string representation of this string.
+        '''Return a string representation of this string. Rounds results to
```

Comment:
This truncates instead of rounding, doesn't it? Assuming that truncation is the correct behavior, you should make one of your test cases something where the truncated value is different from the rounded value.